estate/security/ir.model.access.csv

```diff
@@ -1,2 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_estate_property,access_estate_property,estate.model_estate_property,base.group_user,1,1,1,1
\ No newline at end of file
+estate.access_estate_property,access_estate_property,estate.model_estate_property,base.group_user,1,0,0,0
+estate.access_estate_property_manager,access_estate_property_manager,estate.model_estate_property,estate.group_user,1,1,1,1
```

Comment:
Je pense que tu t'es trompé de groupe ici
```suggestion
estate.access_estate_property_manager,access_estate_property_manager,estate.model_estate_property,estate.estate_manager,1,1,1,1
```

Reply:
oui oui j'ai deja solve this dans un autre commit, suis a jour en local